src/libraries/System.Diagnostics.Tracing/tests/TrimmingTests/EventSourcePropertyValueTest.cs

```diff
@@ -34,6 +35,7 @@ private class TestEventSource : EventSource
         public TestEventSource() : base(EventSourceSettings.EtwSelfDescribingEventFormat) { }
 
         [Event(1)]
+        [DynamicDependency(DynamicallyAccessedMemberTypes.PublicProperties, typeof(TestSubData))]
```

Comment:
I assume this worked with ILLink because the property was accessed in IL (both the getter and the setter).

Reply:
Yep, calling the getter or setter keeps the property right now.

The property could still be removed in theory since what is used is just getter/setter method, but illink never does it right now (it could do it if DebuggerSupport=false, since it makes debugging sub-optimal). It's what I called "lucky IL trimming implementation details".